.github/workflows/build-common.yml

```diff
@@ -44,8 +63,6 @@ jobs:
       fail-fast: false
       matrix:
         os:
-          - macos-latest
-          - macos-13
```

Comment:
we don't test macos in instrumentation repo, I think its probably ok not to test in this repo(?)